scripts/fledge

```diff
@@ -636,6 +637,116 @@ fledge_purge() {
     source "$FLEDGE_ROOT/scripts/storage" purge
 }
 
+# Perform a healthcheck on the Fledge instance
+fledge_healthcheck() {
+
+	result=`fledge_status "silent"`
+	if [[ $result != "0" ]]; then
+		fledge_log "info" "Fledge appears not to be running, healthcheck should be performed on a running system . Start Fledge first." "all" "pretty"
+		exit 0
+	fi
+
+	echo Fledge Healthcheck
+	echo ==================
+	echo
+
+	auth="optional"
+	result=`curl -k -s ${REST_API_URL}/fledge/health/logging`
+
+	if [[ "${result}" == "401"* ]]; then
+		token=`fledge_authenticate`
+		if [[ "${token}" =~ "failed" ]]; then
+			fledge_log "info" "Failed authentication when attempting to stop fledge." "all" "pretty"
+			echo "Authentication failed."
+			exit 0
+		fi
+		result=`curl -k -s -H "authorization: $token" ${REST_API_URL}/fledge/health/logging`
+		auth="required"
+	fi
+
+	a=`echo $result | jq '.levels[] | {name,level}' | jq 'select (.level == "info") .name'`
```

Comment:
Can we avoid `jq` here? Otherwise we need to add this APT package in Debian Packaging.

Reply:
It's a lot easier with it. I thought we already had it. If not we could do it with some horrid python scripting!